docs/docs/0.5.0/policies/health-check.md

```diff
@@ -1,51 +1,31 @@
 # Health Check
 
-The goal of Health Checks is to minimize the number of failed requests due to temporary unavailability of a target endpoint.
-By applying a Health Check policy you effectively instruct a dataplane to keep track of health statuses for target endpoints.
-Dataplane will never send a request to an endpoint that is considered "unhealthy".
+This policy enables Kuma to keep track of the health of every data plane proxy, with the goal of minimizing the number of failed requests in case a data plane proxy is temporarily unhealthy.
```

Comment:
We have so many different spellings across the docs: data plane, data-plane, dataplane and Dataplane :)